bokeh/tile_providers.py

```diff
@@ -1,59 +1,109 @@
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
The standardized boilerplate across the project does not have a space between the `#` and the first `-`, so these changes should be removed throughout.

Reply:
Oh damn I guess that was my auto-pep8. Will do.